packages/terra-tag/README.md

```diff
@@ -0,0 +1,26 @@
+# Terra Tag
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-tag.svg)](https://www.npmjs.org/package/terra-tag)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+The tag component is used for tagging and provides users a way to trigger actions in the UI. Each tag should have a text.
```

Comment:
"Each tag should have a text" needs to be rephrased. Maybe, "Each tag should have text"?

Reply:
Fixed it.